cli/internal/client/client.go

```diff
@@ -30,21 +32,55 @@ func (api *ApiClient) SetToken(token string) {
 
 // New creates a new ApiClient
 func NewClient(baseUrl string, logger hclog.Logger, turboVersion string) *ApiClient {
-	return &ApiClient{
-		baseUrl:      baseUrl,
-		turboVersion: turboVersion,
+	client := &ApiClient{
+		baseUrl:          baseUrl,
+		turboVersion:     turboVersion,
+		CurrentFailCount: 0,
```

Comment:
Based on the [uber style guide](https://github.com/uber-go/guide/blob/master/style.md#omit-zero-value-fields-in-structs) I would leave out this particular field, unless there's a comment for why we're explicitly setting it to `0`, which would happen automatically.